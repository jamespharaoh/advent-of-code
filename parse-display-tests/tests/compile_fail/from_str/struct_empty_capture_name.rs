use parse_display::FromStr;

#[derive(FromStr)]
#[from_str(regex = "(?P<>)")]
struct TestStruct;

fn main() {}
