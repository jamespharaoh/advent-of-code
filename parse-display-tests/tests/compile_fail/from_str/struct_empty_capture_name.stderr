error: `(?P<>)` (empty capture name) is not allowed in struct's regex.
 --> $DIR/struct_empty_capture_name.rs:4:20
  |
4 | #[from_str(regex = "(?P<>)")]
  |                    ^^^^^^^^
